File: Aulas PA_JavaScript/java & outros/notasfor.cpp
#include <iostream>
class Calculo {
public:
    void calcularMedia() {
        int soma = 0;
        int nota;
        for (int i = 0; i < 4; i++) {
            std::cout << "Insira a nota " << (i + 1) << ": ";
            std::cin >> nota;
            
            while (nota < 0 || nota > 10) {
                std::cout << "Nota inválida. Insira a nota " << (i + 1) << " novamente: ";
                std::cin >> nota;
            }
            soma += nota; 
        }
        double media = static_cast<double>(soma) / 4; 
        std::cout << "A média é: " << media << std::endl; 
    }
};
int main() {
    Calculo calculo;
    calculo.calcularMedia();
    return 0;
}

File: Aulas PA_JavaScript/java & outros/Notasfor.java
import java.util.*;

public class Notasfor{
    public static void main(String[]args){
        Scanner sc = new Scanner(|System.in);

        double somanotas = 0.0;
        double notas = 0.0;
        double media = 0.0;

        for(int i = 0; i < 4; i++){
            System.out.println("insira um número: "+(i+1)+":")
            somanotas += notas;
            if(notas <0 && >10){
                System.out.println("inválido");
            }
        }
        media = notas /4;
        System.out.println("sua média é: ", media);
    }
    sc.close();
}